vulture/core.py

```diff
@@ -191,8 +205,9 @@ def file_lineno(item):
         for item in sorted(
                 self.unused_funcs + self.unused_imports + self.unused_props +
                 self.unused_vars + self.unused_attrs, key=file_lineno):
-            print("%s:%d: Unused %s '%s'" % (
-                format_path(item.filename), item.lineno, item.typ, item))
+            print("%s:%d: Unused %s '%s' (%s%% sure)" % (
+                format_path(item.filename), item.lineno, item.typ, item,
+                item.confidence))
```

Comment:
I am not sure if we would want to format the output this way, does this looks good?